### ISSUE TITLE
Update GCP icons

### ISSUE THREAD

**Issue by mikail-khan** [jgraph/drawio#3066]
Missing GCP icons:

- Vertex AI
- Google Cloud Directory Sync
- Deploy
- Identity

Please could you add those.

**Comment by davidjgraph** [comment]
Are those icons in the official google set?

**Comment by mikail-khan** [comment]
Just to switch it around: what's your internal process to keep up to date with GCP? GCP, as you may know, is a constantly evolving platform; services and products are continually being deprecated and added. AT what frequency do you update draw.io to sync with what's on GCP?

**Comment by davidjgraph** [comment]
We check the official set monthly.

**Comment by mikail-khan** [comment]
OK. Have you come across this library:

https://cloud.google.com/icons

Could you do a reconciliation to see which icons you currently don't have in draw.io and which ones possibly need updating in your next round of monthly checks?

**Comment by mikail-khan** [comment]
I don't think this ticket should be closed. You seem to think it's not important that these icons are not available in draw.io. It's highly inconvenient those icons are part of the current selection. Trying to handcraft them and get the same look and feel as the others is takes a fair bit of effort and even then the result is not good. Can you at least share when your next monthly review is scheduled for? Also it would be worth keeping this open so you know which ones I have spotted that need attention.

**Comment by ducktropical** [comment]
I do think that it should not be closed too. It's sad that those icons are not present with the importance that Vertex AI is taking.

**Comment by davidjgraph** [comment]
We can't have a ticket open all the time because there might be new icons. If there are new icons, please open a new ticket, which will be closed when updated.
